app/policies/application_policy.rb

```diff
@@ -36,6 +36,10 @@ def manage?
     update? && record.published
   end
 
+  def subscriptions?
+    true
```

Comment:
this should probably use the same logic as the auth logic that enables subscriptions, not sure how it's done right now